README.md

```diff
@@ -14,66 +14,59 @@ Takeaway Challenge
 
  ```
 
-Instructions
--------
+This repository contains the week 2 Makers Academy homework challenge. The program is run on the command line and allows you to see everything on a takeaway menu, place orders from that menu and complete the order and have a message confirming your order and a delivery time sent to your phone using Twilio.
```

Comment:
Good detail in the README.
Could have added a diagram of your application design to provide even more information.
RE commits - I would encourage you to make your commits smaller - make a commit for every change. A change is: writing a test, using TDD to write code until the test passes, refactor if appropriate.